chapters/conclusion.tex

```diff
@@ -1 +1,283 @@
+Chapters~\ref{chp:kmodes}~and~\ref{chp:copd}. Meanwhile, requests were made for
+larger, more detailed datasets --- so that contemporary, machine learning
+techniques could be applied more readily --- and the gathering of literature
+began. Even with the expansive nature of machine learning literature, two clear
+patterns emerged.
+
+First, the vast majority of publications that introduced a novel machine
+learning method contained a boilerplate evaluation section. In each article, the
+proposed method would be pitted against a few of its contemporaries by comparing
+a handful of metrics on a handful of datasets. Typically, these metrics and
+datasets would be taken from a small pool which was relevant to that technique;
+this makes sense. There are appropriate measures for various techniques, and it
+is important to make comparisons relative to a fixed point.
+
+However, this approach to algorithm evaluation exhibited two issues: one, a lack
+of diversity in the resources used to assess algorithm performance; and, two, an
+incongruence between the power of the evaluation process and the conclusions
+drawn from that process. Often, a method would be deemed `state-of-the-art' or
+`better' based exclusively on a process that offered little insight into the
+actual quality of that method. This reliance on a narrow assessment process
+prompted research into how else an algorithm could be evaluated objectively,
+directly leading to the work in Chapter~\ref{chp:edo}.
+
+The second observation was that the ethics of data and machine learning
+algorithms were being discussed, but the discourse appeared separately from the
+machine learning publications. An exception to this was the development fair
+machine learning practices. These methods consider new formulations and
+objectives based on mathematical fairness, a concept either derived from or with
+common roots in game theory. The reinventing of techniques and paradigms to be
+fair raised questions about how far an addition of some game theory could
+improve the performance of an existing algorithm, as opposed to creating an
+entirely new one, leading to the work in Chapter~\ref{chp:kmodes}.
+
+Over the course of this project, it became clear that the requests for more
+detailed datasets would not complete in time. Hence, the research methodology
+would have to prioritise extracting as much useful information from the data at
+hand. Simultaneously, \ctmuhb\ provided further administrative datasets which
+related to the members of their population who suffer from the respiratory
+condition, COPD. An internal report by NHS Wales found that \ctmuhb\ had the
+highest prevalence of the condition out of all the Welsh health boards. Given
+the known links between COPD, socioeconomic deprivation and the coincidence of
+multiple comorbidities, the objective of this project was revised to focus on
+understanding the needs of that population.
+
+Each of these concerns culminated in the methodology presented in
```

Comment:
I wonder if we can use a softer word than "concerns", perhaps "challenges"?

```suggestion
Overcoming each of these challenges culminated in the methodology presented in
```

Reply:
A good point well made. I'll make these changes now 👍